ui/js/directives/treeherder/resultsets.js

```diff
@@ -39,17 +39,11 @@ treeherder.directive('thAuthor', function () {
     return {
         restrict: "E",
         link: function (scope, element, attrs) {
-            var userTokens = attrs.author.split(/[<>]+/);
-            var email = "";
-            if (userTokens.length > 1) {
-                email = userTokens[1];
-            }
-            scope.authorName = userTokens[0].trim();
-            scope.authorEmail = email;
+            scope.authorEmail = attrs.author.split(/[<>]+/)[0].trim();
```

Comment:
The variable name implies this will be the email, but the first token is actually the author's name. Could you change one or the other? Presuming we want to stick to name?